dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/AnnotationsBetaHelper.java

```diff
@@ -33,9 +77,11 @@
      * product edition is set to "EARLY_ACCESS".  If available, the product
      * "com.ibm.websphere.appserver" is tested.  If that product is not
      * available, the product "io.openliberty" is tested instead.
+     *
+     * Post beta, the product edition test is no longer used.
      */
-    public static final boolean IS_LIBERTY_BETA_PRODUCT =
-        setIsLibertyBetaProduct();
+    public static final boolean IS_LIBERTY_BETA_PRODUCT = true;
+    //     setIsLibertyBetaProduct();
 
     // TODO: This is quite expensive, especially if there are multiple
```

Comment:
What is this TODO comment referring to? Can these be removed?

Reply:
The TODO is for loading the product information, which does disk reads of the product files to load the edition (and other properties).  That TODO is obsolete since the code that loads the product files is no longer invoked.  That code could be removed, but, see the text of the following reply.